engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/inputSettings/ChangeBindingPopup.java

```diff
@@ -55,7 +57,7 @@
 
     private UIInputBind bindButton;
 
-    private BindsConfig defaultBinds;
+    private BindsConfig defaultBinds, currBinds;
```

Comment:
Please change this to

```java
private BindsConfig defaultBinds;
private BindsConfig currBinds;
```

It's okay to declare multiple local variables on a single line, but that's not a great idea for class fields.